### PR TITLE
Portworx E2E changes

### PR DESCRIPTION
**Pull request by mumutyal**

### CODE REVIEW

**Comment by akgunjal** [comment]
@mumutyal : Please create a `tests` folder at root level and inside create `e2e` folder (small case) and there you add your code.
Refer https://github.ibm.com/alchemy-containers/armada-storage-block-plugin/tree/develop/tests/e2e and keep similar folder structure.

**Review by narkarum — COMMENTED** [review]
rename the folder as e2e.

**Comment by narkarum** [comment]
Update the readme.

**Review by narkarum — CHANGES_REQUESTED** [review]
Update readme

**Review by narkarum — COMMENTED** [review]
I see some hardcoded values..are they fixed?

**Review by akgunjal — APPROVED** [review]
Approved for the test case flow.

File: tests/e2e/e2e-tests/block_volume_attach.go [hunk 1]
```diff
@@ -143,6 +138,19 @@ func fileExists(filename string) (bool, error) {
...
 }
 
+func getAttchStatus() (string, error) {
+	attachStatus := "attaching"
+	err := errors.New("Timed out in PV creation")
+	for start := time.Now(); time.Since(start) < (10 * time.Minute); {
+		attachStatus = pv.ObjectMeta.Annotations["ibm.io/attachstatus"]
+		if attachStatus == "attached" || attachStatus == "failed" {
+			return attachStatus, nil
+			break
```
Comment (akgunjal) [inline tests/e2e/e2e-tests/block_volume_attach.go]: Remove this break as its never called since you return from previous line

Comment (mumutyal) [reply tests/e2e/e2e-tests/block_volume_attach.go]: ok

**Comment by narkarum** [comment]
Please check the tests once.

**Comment by narkarum** [comment]
LGTM

**Review by narkarum — APPROVED** [review]

**Pull request merged**